tf2onnx/onnx_opset/tensor.py

```diff
@@ -1248,3 +1248,142 @@ def version_10(cls, ctx, node, **kwargs):
         target_dtype = TensorProto.INT64
         if seq_len_dtype != target_dtype:
             ctx.insert_new_node_on_input(node, "Cast", node.input[1], to=target_dtype)
+
+
+@tf_op("ReverseV2")
+class ReverseV2:
+    @classmethod
+    def version_9(cls, ctx, node, **kwargs):
+        raise NotImplementedError("ReverseV2 is not supported to convert in OPSET 9,"
+                                  " if possible please try using OPSET>=10 instead.")
+
+    @classmethod
+    def version_10(cls, ctx, node, **kwargs):
+
+        axes_node = node.inputs[1]
+        axes = axes_node.get_tensor_value() # Is a Python list.
+        len_axes = len(axes)
+
+        # Store input and output parameters of the ReverseV2 node.
+        rv2_in_names = [node.input[0]]
+        rv2_out_name = node.output
+        output_shape = ctx.get_shape(node.output[0])
+        len_shape = len(output_shape)
+        rv2_node_name = node.name
+
+        # Remove ReverseV2 node from graph.
+        ctx.remove_node(rv2_node_name)
+
+        # List to keep track of input names.
+        inputs = [rv2_in_names]
+
+        new_node = None
+
+        # Empty axis vector.
+        if len_axes == 0:
+            # Replace ReverseV2 with an identity block.
+            new_node = ctx.make_node(
+                "Identity",
+                inputs=inputs[-1],
+                outputs=rv2_out_name,
+                op_name_scope=rv2_node_name,
+            )
+
+        else:
+            # For negative indices use the positive counterpart.
+            for i, ax in enumerate(axes):
+                if ax < 0:
+                    axes[i] += len_shape
+            
+            axes = sorted(axes)
+
+            orig_perm = list(range(len_shape))
+            curr_perm = []
+
+            # Add ReverseSequence nodes for each element of axis.
+            for i in range(len_axes):
+
+                axis = axes[i]
+
+                curr_perm = orig_perm.copy()
+                # Permutation indices relative to original tensor.
+                curr_perm[axis], curr_perm[0] = curr_perm[0], curr_perm[axis]
+
+                # Add a Transpose node if the axis != 0 (finish first due to sort).
+                if axis != 0:
+                    # Permutation indices for the transpose node relative to IN tensor shape.
+                    new_node = ctx.make_node(
+                        "Transpose",
+                        inputs=inputs[-1],
+                        op_name_scope=rv2_node_name,
+                        outputs=None,
+                        attr={"perm": curr_perm}
+                    )
+
+                    inputs.append([new_node.output[0]])
+
+                # Add a Constant node (seq_len) for ReverseSequence.
+
+                # Index 1 for the shape should not return 0
+                # since the input must have rank >= 2.
+                rs_batch_size = output_shape[1] if new_node is None \
+                    else ctx.get_shape(new_node.output[0])[1]
```

Comment:
just use `rs_batch_size = ctx.get_shape(inputs[-1][-1])[1]`? then no need to sort axes.